### PR TITLE
version 2

### PR DESCRIPTION
**Pull request by gsmcmurtrie**
Changes include:
Removing "Get Historical Data" action as this was exposing a Web API end-point marked as obsolete.

Correcting a number of the defined response schemas that had content nested under a "datasource" element.

Updating action descriptions and markdown readme.

Adding required meta data to swagger API definition.

---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

**Comment by sriyen-msft** [comment]
@gsmcmurtrie Please sign the CLA

File: certified-connectors/Industrial App Store/apiDefinition.swagger.json [hunk 1]
```diff
@@ -330,63 +330,57 @@
...
                   "type": "object",
                   "properties": {
-                    "tagname": {
+                    "TagName": {
+                      "type": "string",
+                      "description": "TagName"
+                    },
+                    "UtcSampleTime": {
+                      "type": "string",
+                      "description": "UtcSampleTime"
```
Comment (sriyen-msft) [inline certified-connectors/Industrial App Store/apiDefinition.swagger.json:342]: @gsmcmurtrie Can you please provide valid description values.

Documentation - [Summary and Description values](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#operation-and-parameter-summaries-and-descriptions)

Applies to all the description values in the swagger

File: certified-connectors/Industrial App Store/apiProperties.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
         "type": "oauthSetting",
         "oAuthSettings": {
           "identityProvider": "oauth2",
-          "clientId": "<<Please add your clientID here>>",
+          "clientId": "18e880e61fe84e9485fda4eb41462554",
```
Comment (sriyen-msft) [inline certified-connectors/Industrial App Store/apiProperties.json:8]: @gsmcmurtrie You can replace the clientId value with `[[Dummy]]`

File: certified-connectors/Industrial App Store/apiProperties.json [hunk 3]
```diff
@@ -28,7 +28,9 @@
         }
       }
     },
-    "iconBrandColor": "#CCCCCC",
-    "capabilities": []
+    "iconBrandColor": "#ffffff",
```
Comment (sriyen-msft) [inline certified-connectors/Industrial App Store/apiProperties.json:31]: @gsmcmurtrie Please note that white background color for icon is not allowed.

Documentation - [Icon](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#icon)

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by ptalrejagithub** [comment]
@gsmcmurtrie You have submitted the intro.md instead of readme.md in open repo. There is a difference between intro.md and readme.md files, the intro.md file is required for certification and its content would be published and displayed in your connector's documentation page. Readme.md file is only required for open sourcing your connector which can be helpful to the open source users who may want to contribute in your connector. Here is an [example](https://github.com/st0014/PowerPlatformConnectors/blob/dev/custom-connectors/AzureKeyVault/Readme.md)

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**